sklearn/utils/fixes.py

```diff
@@ -343,6 +343,11 @@ def __call__(self, *args, **keywords):
     from functools import partial
 
 
+def parallel_helper(obj, methodname, *args, **kwargs):
```

Comment:
I'm not sure about putting this in `fixes`. Exporting it to fixes decreases legibility, given how simple it is. Ambivalent. @GaelVaroquaux may have a strong opinion, though.